test/ios/mocked_data/test_get_bgp_neighbors_detail/normal/show_ip_bgp_ipv6_unicast_neigh_2001_559__1.txt

```diff
@@ -3,7 +3,7 @@ BGP neighbor is 2001:559::1,  remote AS 7922, external link
  Description: Comcast IPv6 Transit
  Inherits from template PEER-SESSION-AS7922 for session parameters
   BGP version 4, remote router ID 68.86.1.1
-  BGP state = Established, up for 12w2d
+  BGP state = Established, up for 12:22:20
```

Comment:
Same comment as https://github.com/napalm-automation/napalm/pull/1185/files#r416569119

Reply:
All understood, I've reverted the the file to the previous version